### PR TITLE
Use octicons for all repo header buttons

### PR DESCRIPTION
**Pull request by CirnoT**
Fixes inconsistent height between fork and other buttons

### CODE REVIEW

File: web_src/less/_repository.less [hunk 1]
```diff
@@ -33,8 +33,8 @@
             white-space: nowrap;
         }
 
-        .svg.octicon-repo-forked {
-            margin-top: -1px;
+        .svg {
+            margin: 0 .42857143em 0 -.21428571em;
```
Comment (silverwind) [inline web_src/less/_repository.less:36]: Why these strange values? Are they based on fomantic's?

Comment (CirnoT) [reply web_src/less/_repository.less:36]: Yes

Comment (CirnoT) [reply web_src/less/_repository.less:36]: Additionally this fixes lack of margin on right side between octicon and text by using same margin as icons used.

Comment (silverwind) [reply web_src/less/_repository.less:36]: Ok. I'd try to avoid negative margins if possible, thought.

Comment (CirnoT) [reply web_src/less/_repository.less:36]: I'd prefer to use Fomantic styling here for consistency sake.

**Comment by CirnoT** [comment]
Pushed small fix to ensure that margin isn't applied to all svgs on repo header

**Review by silverwind — APPROVED** [review]

**Review by gary-kim — APPROVED** [review]

**Review by lafriks — APPROVED** [review]

**Pull request merged**